common/src/main/java/com/sonicether/soundphysics/config/SoundAbsorptionConfig.java

```diff
@@ -0,0 +1,114 @@
+package com.sonicether.soundphysics.config;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.Map;
+
+import com.sonicether.soundphysics.Loggers;
+
+import com.sonicether.soundphysics.integration.voicechat.AudioChannel;
+import de.maxhenkel.configbuilder.CommentedProperties;
+import de.maxhenkel.configbuilder.CommentedPropertyConfig;
+import net.minecraft.core.registries.BuiltInRegistries;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.sounds.SoundEvent;
+import net.minecraft.sounds.SoundEvents;
+
+public class SoundAbsorptionConfig extends CommentedPropertyConfig {
```

Comment:
This and the reflectivity config should have shared code via an abstract parent class